### ISSUE TITLE
How can I get specific user emails using API?

### ISSUE THREAD

**Issue by umar13893** [mail-in-a-box/mailinabox#2037]
Hi, I am looking to get user emails using API but I could not find this type of API endpoint.

**Comment by Courvix** [comment]
Do you mean retrieving actual email messages?

Chances are if an API call you're looking for isn't in the documentation, the API doesn't have that call.

**Comment by umar13893** [comment]
@Courvix Yes the actual email messages of a specific user like user@domain.com

**Comment by Synchro** [comment]
There is an API – it's called IMAP
